source/stylesheets/modules/_anchored-heading.scss

```diff
@@ -1,14 +1,37 @@
 .anchored-heading {
+  $icon-width: 23px;
+  $icon-height: 14px;
+  $icon-spacing: 3px;
+
   position: relative;
-}
 
-.anchored-heading__icon {
-  position: absolute;
-  display: none;
-  text-decoration: none;
-  left: -1em;
-}
+  &__icon {
+    position: absolute;
+    width: $icon-width;
+    left: -($icon-width + $icon-spacing);
+    padding-right: $icon-spacing;
+    margin-top: -1px;
+
+    opacity: 0;
+
+    text-decoration: none;
+    text-indent: -9999em;
+
+    background-image: image-url('anchored-heading-icon.png');
+    background-repeat: no-repeat;
+    background-position: left center;
+
+    @include device-pixel-ratio {
+      background-image: image-url('anchored-heading-icon-2x.png');
+      background-size: $icon-width $icon-height;
+    }
+  }
+
+  &:hover &__icon {
+    opacity: 1;
 
-.anchored-heading:hover .anchored-heading__icon {
-  display: block;
+    @include media(mobile) {
```

Comment:
Ideally this should be 'mobile first', so `display: none;` and then a `tablet` breakpoint that does `display: inherit` and wraps pretty much all of the rest of the behaviour.